test/unit/utils.html

```diff
@@ -91,12 +90,76 @@
 
       setTimeout(function() {
         assert.equal(called, 1, 'debounce should be called exactly once');
-        assert(Date.now() - now > 100, 'debounce should be called after at least 100ms');
```

Comment:
Is there a reason you took out this assert?

`debounce` supports two modes, one without a timeout, in which case it will run at next microtask, and one with a timeout, in which case it should run no earlier than the timeout (which is what that was verifying).